src/components/AttachmentPicker/index.js

```diff
@@ -34,8 +35,10 @@ class AttachmentPicker extends React.Component {
                         const file = e.target.files[0];
 
                         if (file) {
+                            const cleanName = FileUtils.cleanFileName(file.name);
                             file.uri = URL.createObjectURL(file);
-                            this.onPicked(file);
+                            const updatedFile = new File([file], cleanName);
```

Comment:
I'm wondering if we only want to do this IF the name has been changed, since it won't usually need to be changed. Something like `if (cleanName != file.name) {}`? What do you think, @mananjadhav

Reply:
I agree. We're rarely going to have this condition. So doesn't make sense to create a copy.

Reply:
I've updated the change.